Dockerfile.analytics

```diff
@@ -18,7 +18,8 @@ RUN pip3 install matplotlib==2.0.2 \
     psycopg2==2.7.2 \
     seaborn==0.8 \
     pandas==0.22.0 \
-    https://github.com/better/alpine-tensorflow/releases/download/alpine3.7-tensorflow1.7.0/tensorflow-1.7.0-cp36-cp36m-linux_x86_64.whl
+    https://github.com/better/alpine-tensorflow/releases/download/alpine3.7-tensorflow1.7.0/tensorflow-1.7.0-cp36-cp36m-linux_x86_64.whl \
+    scikit-learn==0.19.1
```

Comment:
Maybe more clean to add official packages before unofficial ones

Reply:
actually my preference is to put them in alphabetical order. i can do that.